cli/command/command.go

```diff
@@ -239,7 +249,7 @@ func Populate(context *project.Context, c *cli.Context) {
 
 	if c.Command.Name == "logs" {
 		context.Log = true
-	} else if c.Command.Name == "up" {
+	} else if c.Command.Name == "up" || c.Command.Name == "create" {
```

Comment:
We really need to fix this context object. It doesn't make sense to have a single context for every command.

I think we should be using something like https://github.com/spf13/cobra to handle subcommands.